libnavui-maneuver/src/main/res/values-night/colors.xml

```diff
@@ -8,7 +8,7 @@
   <color name="mapbox_main_maneuver_background_color">@color/colorSecondary</color>
   <color name="mapbox_sub_maneuver_background_color">@color/colorSecondaryVariant</color>
   <color name="mapbox_upcoming_maneuver_background_color">@color/colorSecondaryVariant</color>
-  <color name="mapbox_exit_text_color">@android:color/black</color>
+  <color name="mapbox_exit_text_color">@color/colorOnSurface</color>
```

Comment:
With a day theme it is going to be #3C3E3E on a #FFFFFF background. 
With a night theme it is going to be #F8F8F8 on a #0D0D0D background.

Reply:
> With a day theme it is going to be #3C3E3E text on a #FFFFFF background.
With a night theme it is going to be #F8F8F8 text on a #0D0D0D background.

This is not the desired behavior at least based on the last designs. The background should be white and text should be black. We can revisit this and discuss with designs, but let's not make that change until then.

@Zayankovsky

Reply:
I don't think you need to add this because the day colors.xml uses the same color name.

Reply:
> This is not the desired behavior at least based on the last designs. The background should be white and text should be black. We can revisit this and discuss with designs, but let's not make that change until then.

In normal scenarios (the standalone view is used without integrating Drop-In or only Drop-In view is used) this is still working according to the designs. I only changed the behavior when Drop-In is integrated, but the standalone view is used for some reason, which already violates these designs in `main`, but at least with my fix it will look fine (no more black text on a black background). The fix is necessary because it affects those using Android Auto and Drop-In in parallel. A proper solution would be to copy all necessary resources from `libnavui-resources` module to `libnavui-androidauto` and give them unique names, so that Drop-In cannot override them, but I wanted to avoid that.